js/views/questionView.js

```diff
@@ -79,6 +80,11 @@ class QuestionView extends ComponentView {
 
   }
 
+  onSubmittedChanged(model) {
+    // Remove error class, if it exists
+    model.toggleClass('has-error', false);
```

Comment:
Could you move this to `onResetClicked`? Is there a reason why you need to do it when `_isSubmitted` changed?

Reply:
Yep, that makes sense. I've made this change.